server/neptune/workflows/internal/deploy/revision/queue/worker.go

```diff
@@ -53,6 +62,40 @@ const (
 	receive  = "receive"
 )
 
+func NewWorker(
+	ctx workflow.Context,
+	q queue,
+	a workerActivities,
+	tfWorkflow terraform.Workflow,
+	repoName, rootName string,
+) (*Worker, error) {
+	tfWorkflowRunner := terraform.NewWorkflowRunner(a, tfWorkflow)
+	deployer := &Deployer{
+		Activities:              a,
+		TerraformWorkflowRunner: tfWorkflowRunner,
+	}
+
+	latestDeployment, err := deployer.FetchLatestDeployment(ctx, repoName, rootName)
```

Comment:
thoughts on just renaming latest deployment to previous deployment for variables and fxns everywhere?

Reply:
or should i rename previousDeployment to latestDeployment?

Reply:
that's fine too